rancher/v1.6/en/cattle/volumes/index.md

```diff
@@ -0,0 +1,208 @@
+
+##### Shared Environment Scope Volume Example
+
+First, [create the environment scope volume]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/cattle/volumes/#create-a-volume) to be shared between stacks.
+
+```
+$ rancher volume create --driver local redis-data-external
+```
+
+Define the volume with `external: true` modifier to indicate the volume's lifecycle is managed externally (environment scope). Save the `docker-compose.yml` file below to a directory.
+
+```yaml
+version: '2'
+services:
+  redis:
+    image: redis:3.0.7
+    volumes:
+    - redis-data-external:/data
+volumes:
+  redis-data-external:
+    driver: local
+    external: true
+```
+
+Now create the stack.
+
+```
+$ rancher stack create
+```
+
+[List the volumes]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/cattle/volumes/#list-volumes) to verify that `redis-data-external` volume is `active`.
+
+> **Note:** Scaling a service up or down mounts or unmounts the same shared volume.
+
+Any new stack within the environment may attach to the same `redis-data-external` volume. The easiest way is to copy the volume definition verbatim.
+
+```yaml
+volumes:
+  redis-data-external:
+    driver: local
+    external: true
+```
+
+#### Stack Scope
+
+A stack-scoped volume is owned and managed by the stack in which it is defined. The primary benefit is the volume lifecycle is automatically managed by Rancher as part of the stack lifecycle.
+
+##### Shared Example
+
+Save the `docker-compose.yml` file below to a directory. Notice that no modifiers are added to the volume; stack scope is default behavior.
+
+```yaml
+version: '2'
+services:
+  redis:
+    image: redis:3.0.7
+    volumes:
+    - data:/data
+volumes:
+  data:
+    driver: local
+```
+
+Now create the stack.
+
+```
+$ rancher stack create
+```
+
+To avoid name collisions with other volumes, unique names are used for the stack scope volumes you define. They include the stack name, original volume name, and a random component. [List the volumes]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/cattle/volumes/#list-volumes) and identify the `data` volume exists in this form.
+
+> **Note:** Stack-scope volumes could be mounted by other stacks. Stack scope is not a security mechanism. It is solely for managing the lifecycle of the volume.
+
+### V1 Compose
+
+Up until this point, we've discussed volumes in the context of [Docker V2 Compose](https://docs.docker.com/compose/compose-file/compose-file-v2/#volumes-volume_driver).
+
+[Docker V1 Compose](https://docs.docker.com/compose/compose-file/compose-file-v1/#volumes-volume_driver) may also be used. Rancher passes V1 volumes directly to Docker; as such, there are no scheduling guarantees and volumes are not automatically deleted. To bridge the gap and ensure containers are scheduled to the host where a V1 volume resides, [scheduling affinities](https://docs.rancher.com/rancher/v1.6/en/cattle/scheduling/#fields-1) may be used.
```

Comment:
I think one sentence up front explaining the syntactic difference might help. LIke:
"...may also be used. In V1 compose, there is no top-level volume section and volumes can only be defined within the context of a service."